### ISSUE TITLE
Detection

### ISSUE THREAD

**Issue by Onyxoulis** [UnamSanctam/UnamBinder#47]
Detected from Google Chrome

**Comment by UnamSanctam** [comment]
Yes that's quite normal, unless you try to make it undetected from Googles antivirus. There's also other methods such as zipping the file with a password and using that.
